docs/checks.md

```diff
@@ -517,7 +517,7 @@ information about a bug is not publicly visible.
 
 **Remediation steps**
 - Place a security policy file `SECURITY.md` in the root directory of your repository. This makes it easily discoverable by a vulnerability reporter.
-- The file should contain information on what constitutes a vulnerability and a way to report it securely (e.g. issue tracker with private issue support, encrypted email with a published public key).
+- The file should contain information on what constitutes a vulnerability and a way to report it securely e.g., an issue tracker with private issue support, encrypted email with a published public key. Follow the [coordinated vulnerability disclosure guidelines](https://github.com/ossf/oss-vulnerability-guide/blob/main/guide.md) to respond to vulnerability disclosures.
```

Comment:
Tiny nit: if you're going to take the examples out of the parentheses, just be sure to put a comma before it.

Reply:
> Tiny nit: if you're going to take the examples out of the parentheses, just be sure to put a comma before it.

hehe, yep: https://github.com/ossf/scorecard/pull/1545#discussion_r793877212